compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaSymbol.java

```diff
@@ -65,7 +68,21 @@ protected BallerinaSymbol(String name, SymbolKind symbolKind, BSymbol symbol, Co
 
     @Override
     public Optional<String> getName() {
-        return Optional.ofNullable(this.name);
+        // In the langlib context, reserved keywords can be used as regular identifiers. Therefore, they will not be
+        // escaped.
+        if (this.escapedName != null) {
+            return Optional.of(this.escapedName);
+        }
+        if (getModule().isPresent()) {
+            ModuleID moduleID = getModule().get().id();
+            if (moduleID != null && moduleID.moduleName().startsWith("lang")
```

Comment:
The `moduleID != null` check is also unnecessary since it's expected to be not null.

Reply:
Removed it.